packages/notifications/src/elements/Alert.tsx

```diff
@@ -7,17 +7,36 @@
 
 import React, { HTMLAttributes } from 'react';
 import PropTypes from 'prop-types';
-import { StyledAlert, IStyledAlertProps } from '../styled';
-import { ARRAY_VALIDATION_TYPE } from '../utils/types';
+import { StyledAlert, StyledIcon } from '../styled';
+import { validationIcons, validationHues } from '../utils/icons';
+import { ARRAY_VALIDATION_TYPE, VALIDATION_TYPE, VALIDATION_HUE } from '../utils/types';
+import { NotificationsContext } from '../utils/useNotificationsContext';
+
+export interface IAlertProps {
+  type: VALIDATION_TYPE;
```

Comment:
The API lists `type` as `VALIDATION_TYPE`. The type strings need to be listed instead: https://henry-next.netlify.com/notifications/#alert. No one will know what `VALIDATION_TYPE` means.